requirements-listener.txt

```diff
@@ -10,7 +10,7 @@ msgpack==0.6.1
 prometheus_client==0.6.0
 psycopg2-binary==2.8.1
 pytz==2019.1
-pyyaml==3.13
+pyyaml==5.1.1
```

Comment:
I don't think this will work.  The insights-core component REQUIRES pyyaml version <= 3.13.  I believe this will fail during build unless insights-core team updates their component, and they have indicated they will not do that.